x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/dimension_editor.tsx

```diff
@@ -1157,6 +1157,9 @@ export function DimensionEditor(props: DimensionEditorProps) {
                 columnId={props.columnId}
```

Comment:
I'm probably missing something obvious, but isn't like 1154 already taking care of this here as it's only rendering `BucketNestingEditor` if `hideGrouping` is falsy?